src/settings.js

```diff
@@ -1402,9 +1414,11 @@ var ELIMINATE_DUPLICATE_FUNCTIONS_PASSES = 5;
 var EVAL_CTORS = 0;
```

Comment:
`EVAL_CTORS` only works with the fastcomp backend.

Reply:
Yeah.. sadly.   Although it did used to work and we do plan on making it work again in the future.  Since this is the case I'm not sure we want to warn when its used.